pipeline/README.md

```diff
@@ -4,20 +4,20 @@
 
 ## Bootstrapping
 
-Initial install for a cluster must happen in fybrik-system (currently).  It won't hurt anything if you aren't sure, and reinstall in fybrik-system.
-```
-. source-external.sh
-bash -x bootstrap-pipeline.sh fybrik-system
-# follow on screen instructions
-```
+Run the following command to bootstrap a Fybrik instance. By default the instance of Fybrik will be deployed as namespaced scoped, meaning the controller will only watch and process FybrikApplications from a specific namespace. Therefore limiting you to create FybrikApplications from one namespace. By default that namespace is the one specified on the bootstrap script (a.k.a *system* namespace). You can change this behavior by setting environment variables. For details see [Scoping the deployment](#scoping-the-deployment).
```

Comment:
@laurieaw63 is the default `"FybrikApplications from a specific namespace" ` applies only to when installing via the pipeline? is there an impact on the quick-start documentation?

Reply:
It only applies to installing via pipeline. If manually installing the default will be to accept FybrikApplication from all namespaces.